CHANGES.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 0.4.2
+
+- Convert mistral workflows to orquesta
```

Comment:
Because it's not just a bugfix patch, can we bump the version to `0.5.0` instead?

Reply:
Of course. Done!